.github/workflows/fetch_filter_resources.yaml

```diff
@@ -0,0 +1,121 @@
+name: Weekly resource fetching and community filtering
+
+on:
+  workflow_dispatch:
+  schedule:
+    #Every Sunday at 8:00 am
+    - cron: "0 8 * * 0" 
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "tools"
+  cancel-in-progress: false
+
+jobs:
+  fetch-servers:
+    runs-on: ubuntu-20.04
+    name: Merge tools, fetch tutorials and filter the resources for communities
+      steps:
+      - name: Checkout main
+        uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install requirement
+        run: |
+          python -m pip install -r requirements.txt
+          sudo apt-get install jq
+      - name: Fetch list of all available servers
+          run: | 
+            python sources/bin/get_public_galaxy_servers.py -o sources/data/available_public_servers.csv
+      - name: Archive available servers
+        uses: actions/upload-artifact@v4
+        with:
+          name: available-servers
+          path: sources/data/available_public_servers.csv
+  fetch-tools-stepwise:
+    runs-on: ubuntu-20.04
+    name: Fetch tool stepwise
+    strategy:
+      max-parallel: 1 #need to run one after another, since otherwise there is a chance, that mulitple jobs want to push to the results branch at the same time (which fails due to merge)
```

Comment:
```suggestion
      #max-parallel: 1 #need to run one after another, since otherwise there is a chance, that mulitple jobs want to push to the results branch at the same time (which fails due to merge)
```